packages/dx-react-chart/docs/guides/animation.md

```diff
@@ -1,5 +1,19 @@
 # React Chart - Animation
 
-You can add the [Animation](../reference/animation.md) plugin to the Chart component to animate all the series when they first appear on-screen. To modify animation options, implement a custom point component for a series. In the following example, the duration of each bar's animation is configured individually:
+You can add the [Animation](../reference/animation.md) plugin to the Chart component to animate all the series.
 
-.embedded-demo({ "path": "chart-basic/animation", "showThemeSelector": true })
+## Basic Setup
+
+Import the `Animation` plugin.
+
+## Animation setups
+
+You can modify animation easing and duration. For that use `duration` and `easing` properties in the `Animation` plugin. The following example demonstrates how to change easing using the d3 library and how to change duration:
+
+.embedded-demo({ "path": "animation/custom-easing", "showThemeSelector": true })
+
+## Transition animation
+
+In the following example shows animation when data updated:
+
+.embedded-demo({ "path": "animation/with-transition", "showThemeSelector": true })
```

Comment:
I think this demo should be without bounce effect and display only basic animation.